packages/storybook-config/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@webstudio-is/storybook-config",
+  "private": true,
+  "version": "0.0.0",
```

Comment:
side note: your `@webstudio-is/jest-config` (and other packages like that) are currently public and published to npm. They don't have to be.